src/net.cpp

```diff
@@ -1768,6 +1768,20 @@ int CConnman::GetExtraOutboundCount()
     return std::max(nOutbound - m_max_outbound_full_relay, 0);
 }
 
+int CConnman::GetExtraBlockRelayCount()
+{
+    int nBlockRelay = 0;
```

Comment:
`nBlockRelay` doesn't seem to follow the code conventions we have for vars

Reply:
Fixed.